guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,51 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
```

Comment:
I like "secondary" in favor of "other", but your call. Checking the currently built docs did not help me make a recommentation, though I'd appreciate consistency.

Out of scope & your call.

Reply:
Elsewhere, we are using "disaster recovery site/server" and "primary site/server". I think that could work pretty well. +1 on consistency, that's something I definitely should look into as well.

Reply:
You know what, looking at it more closely now, this probably won't work after all. This is a scenario for `two active servers`. They should both be more or less equal to each other (with the exception of how to ensure content sync, as described in this procedure). So I'll stick with "other" here.